src/js/messaging/containers/Folder.jsx

```diff
@@ -334,33 +335,48 @@ export class Folder extends React.Component {
           this.props.fetchFolder(lastRequest.id, lastRequest.query);
         };
 
-        return (
-          <p>
-            Could not retrieve the folder.&nbsp;
-            <a onClick={reloadFolder}>Click here to try again.</a>
-          </p>
+        componentContent = (
+          <div className="columns">
```

Comment:
@U-DON it's a global class that adds side padding—usually used with column size classes like `small-12`, etc

Without this the content is up against the edges on mobile